content/en/references/init-hooks.md

```diff
@@ -33,7 +33,7 @@ In these directories, you can put either executable shell scripts or Python prog
 All except `boot.d` will be run in the same Python interpreter as LocalStack, which gives additional ways of configuring/extending LocalStack.
 You can also use subdirectories to organize your init scripts.
 
-Currently, known script extensions are `.sh`, and `.py`.
+Currently, known script extensions are `.sh`, `.py`, and `.tf`.
```

Comment:
I would not suggest to mention `.tf` here since it requires installation of an Extension and does not work out of box. You can mention that `.tf` works with the help of an Extension which needs to be installed separately.